languages/zh-CN.yml

```diff
@@ -38,7 +38,6 @@ post:
     post_link: 原文链接
     license_title: 版权声明
     license_content: "本博客所有文章除特别声明外，均采用 %s 许可协议。转载请注明出处！"
-    license_content_reprint: "本文章为转载文章，已获转载许可。转载请注明出处！"
```

Comment:
Why delete the Chinese translation text

Reply:
We use Crowdin to manage translations: https://crowdin.com/project/hexo-theme-next
You just need to add translations here, and it will be synced to the GitHub repository